clients/cobol-dialect-api/src/v1api.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2025 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+import * as vscode from "vscode";
+import { type CopyStatementParser, MainExtensionId } from "./common";
+
+let v1Api: V1Api | undefined = undefined;
+
+/**
+ * Retrieve Version 1 of the API
+ * @param id id of the extension requesting v1 API
+ */
+export const getV1Api = async (extensionId: string): Promise<V1Api | Error> => {
+  if (!v1Api) {
+    const main = vscode.extensions.getExtension<unknown>(MainExtensionId);
+    if (main === undefined) {
+      return new Error(
+        `Cannot find COBOL Language Support extension ${MainExtensionId}`,
+      );
+    }
+    const api = await main.activate();
+    if (
+      typeof api !== "object" ||
+      api === null ||
+      !("version" in api) ||
+      typeof api.version !== "string"
+    ) {
+      return new Error(
+        `The version ${main.packageJSON.version} of COBOL Language Support extension ${main.id} you have installed does not provide API.`,
```

Comment:
Is that meant to tell the user "Update the extension"?
I would prefer something like: "Update to the latest version of COBOL Language Support to use API features", here and below. No need to pull the version number from package.json

Reply:
Fixed